### DISCUSSION TITLE
Does webdriverio support xpath axes ?

### DISCUSSION THREAD

**Discussion by Siizy** [webdriverio/webdriverio#6360] [Community]
I am getting invalid xpath when I use xpath axes methods to locate an element. Just want to know if it is supported.
Thanks

**Comment by mgrybyk** [comment]
Yes, it supported.

**Reply by Siizy** [reply]
Thanks for the reply @mgrybyk . I wrote an xpath with a combination of parent and following-sibling , the chrome detects the correct element but webdriverio says its an invalid xpath. What could be the reason?

**Reply by mgrybyk** [reply]
Share the xpath
